website/src/app/kb/deploy/sites/readme.mdx

```diff
@@ -1,17 +1,33 @@
+import PlanBadge from "@/components/PlanBadge";
+import SupportOptions from "@/components/SupportOptions";
+import NextStep from "@/components/NextStep";
+
+<PlanBadge plans={["starter", "enterprise"]}>
+
 # Sites
 
+</PlanBadge>
+
 Sites represent a shared network environment that gateways and resources exist
-in. All gateways and resources in a site should be able to reach each other.
+within. All gateways and resources in a site need unobstructed network
+connectivity to each other. This is typically achieved by placing all resources
+and gateways in the same physical location or virtual network.
+
+## What is a site?
+
+A site is a collection of resources and gateways that are logically grouped
+together. A site can be a physical location, a virtual network, or any other
+grouping that makes sense for your organization.
+
+A site can be as small as a single gateway running on a single server managing
+the access to the resources on that server, or as large as an entire data center
+managing access to an entire fleet of servers.
```

Comment:
I think the data center will not have a network environment where everything can access everything, maybe we should align the site explanation closer to VPC or some DevOps term well understood by our users?

Reply:
Sure, a better description here would be "data center rack"

Reply:
Updated to say "data center network managing access to an entire cluster of servers". This will be common for AI workloads.